pkg/operator/ceph/cluster/watcher_test.go

```diff
@@ -231,6 +233,29 @@ func TestHandleNodeFailure(t *testing.T) {
 		},
 	}
 
+	cephfsPV := &corev1.PersistentVolume{
```

Comment:
could you try to list NetworkFence and see if the count is two?

Reply:
Yes, updated, working fine.